packages/accounts-base/accounts_common.js

```diff
@@ -274,15 +282,15 @@ export class AccountsCommon {
     }
 
     // Validate config options keys
-    Object.keys(options).forEach(key => {
+    for (const key of Object.keys(options)) {
```

Comment:
Why use `for of` instead of `forEach`?

Reply:
Better performance according to: https://romgrk.com/posts/optimizing-javascript